test/ode/ode_low_storage_rk_tests.jl

```diff
@@ -312,6 +312,31 @@ sol_new = solve(new_prob_ode_nonlinear_inplace, alg, dt=1.e-4, save_everystep=fa
 @test sol_old[end] ≈ sol_new[end]
 
 
+alg = CKLLDDRK43_2()
+dts = 1 ./ 2 .^(8:-1:4)
+for prob in test_problems_only_time
+  sim = test_convergence(dts, prob, alg)
+  @test sim.𝒪est[:final] ≈ OrdinaryDiffEq.alg_order(alg) atol=testTol
+end
+for prob in test_problems_linear
+  sim = test_convergence(dts, prob, alg)
+  @test sim.𝒪est[:final] ≈ OrdinaryDiffEq.alg_order(alg)+1 atol=testTol
+end
+for prob in test_problems_nonlinear
+  sim = test_convergence(dts, prob, alg)
+  @test sim.𝒪est[:final] ≈ OrdinaryDiffEq.alg_order(alg) atol=testTol
+end
+integ = init(prob_ode_large, alg, dt=1.e-2, save_start=false, save_end=false, save_everystep=false)
```

Comment:
You should also add `adaptive=false` if you want to get the minimal number of allocations. It might be good to test both cases (`adaptive=false` and `adaptive=true`). For `adaptive=false`, the storage requirement should be smaller.